.vscode/tasks.json

```diff
@@ -29,6 +29,7 @@
         "args": [
           "msbuild",
           "-p:UseRoslynAnalyzers=false",
+          "-p:GenerateFullPaths=true",
```

Comment:
What's this change for?

Reply:
Full paths are required for VS Code to have clickable output from build tasks.

Reply:
@jasonmalinowski This is the flag that a user would need for that feedback ticket we assigned you a couple weeks back.

Reply:
@sharwell Which feedback ticket? I recall but can't find one around linked files, but I moved that one because it wasn't the IDE's fault.